djreact/urls.py

```diff
@@ -5,7 +5,6 @@
 urlpatterns = [
     path('api-auth/', include('rest_framework.urls')),
     path('rest-auth/', include('rest_auth.urls')),
-    path('rest-auth/registration/', include('rest_auth.registration.urls')),
```

Comment:
No longer calling djangorestauth registration endpoint directly so we shouldn't expose this URL directly in the API. Must go through new wrapper view